### PR TITLE
Using [NotNull] from Common repo package

### PR DESCRIPTION
**Pull request by Praburaj**
@muratg

### CODE REVIEW

File: src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.IO;
+using Microsoft.Framework.Internal;
```
Comment (muratg) [inline src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs]: Is this required here?  Nothing else seems to have changed in this file.

File: src/Microsoft.Framework.Caching.Distributed/LocalCache.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@
 using System;
 using System.IO;
 using Microsoft.Framework.Caching.Memory;
+using Microsoft.Framework.Internal;
```
Comment (muratg) [inline src/Microsoft.Framework.Caching.Distributed/LocalCache.cs]: Ditto

**Comment by muratg** [comment]
:shipit:

File: src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs (hunk 1 above)
Comment (Praburaj) [inline src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs]: Yes the [NotNullAttribute] in the common package lives in this namespace. That's why I added this namespace after removing the `NotNullAttribute.cs` file in each of the projects.

Comment (muratg) [inline src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs]: :+1:

**Pull request merged**